pipelines/controller/pom.xml

```diff
@@ -31,6 +31,15 @@
   <properties>
     <java.version>17</java.version>
     <spring-boot.version>3.1.3</spring-boot.version>
+    <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
+    <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+    <sonar.jacoco.reportPath>${project.basedir}/target/jacoco.exec</sonar.jacoco.reportPath>
+    <sonar.language>java</sonar.language>
+    <sonar.host.url>http://localhost:9000</sonar.host.url>
+    <sonar.projectName>pipelines-controller</sonar.projectName>
+    <sonar.login>admin</sonar.login>
+    <sonar.password>admin</sonar.password>
+    <sonar.java.binaries>target/classes</sonar.java.binaries>
```

Comment:
This change will only publish the report of the controller module. Can we try creating a aggregate report at the root location and try publishing that instead?